icons/popos.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Pop!_OS</title><path d="M12 0A11.998 11.998 0 0 0 0 12 12 12 0 1 0 12 0ZM9.641 2.918c1.091-.026 1.547.229 2.182.635a4.463 4.463 0 0 1 1.902 2.763c.41 1.846-.127 2.519-.127 2.715-.609 1.294-1.622 2.335-3.043 2.842l1.217 3.172c.228.583.431 1.192.254 1.75-.178.558-.989.736-1.573.127-1.116-1.192-4.871-8.702-5.15-9.26-.279-.558-.584-1.016-.584-1.574.027-.837 1.319-1.7 1.953-2.131.634-.431 1.878-1.014 2.969-1.039Zm-.996 2.311c-.789.022-.359 1.669-.198 2.128.285.814 1.042 2.267 1.6 2.36.092.015.734-.09.734-.889a3.258 3.258 0 0 0-.048-1.217C10.318 6.185 9.255 5.37 8.93 5.277a.878.878 0 0 0-.285-.048Zm10.002 2.662c.304.279.33.582.228.963-.101.38-.457 1.193-.736 1.777l-.709 1.344c-1.37 2.435-1.649 2.689-2.029 2.537-.457-.178-.305-2.614.127-5.582.127-.812.328-1.217.556-1.42.253-.225 1.566-.575 2.563.381Zm-3.17 9.488c-.203.127-.686.127-.965-.102-.54-.442-.268-1.45.432-1.65.989-.307 1.267 1.293.533 1.752Zm-8.881 1.014h10.809c.385 0 .937.258.937 1.013 0 .52-.417.94-.937.94H6.596a.937.937 0 0 1-.938-.94c0-.755.552-1.013.938-1.013Z"/></svg>
```

Comment:
Good find @Mvlprem!
I'm seeing small differences here and there. Ex.
![image](https://github.com/simple-icons/simple-icons/assets/12817388/05f47732-86a0-4986-b588-3cdc6bf8ea32)

Might be a precision issue only.

Reply:
@service-paradis Everything should be good now, In case if you feel like there are lot of unnecessary nodes i did tried to clean them up but removing any of those nodes is resulting in small differences like you pointed above.